package.json

```diff
@@ -55,6 +55,7 @@
     "karma-jasmine": "^1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
     "moment": "^2.21.0",
+    "msal": "^0.2.3",
```

Comment:
we still have hellojs dependency ? refer line 51

Reply:
removing hellojs dependency ...